examples/fuse_examples/imaging/classification/isic/isic_runner.py

```diff
@@ -93,12 +108,12 @@
 TRAIN_COMMON_PARAMS["data.num_folds"] = 5
 TRAIN_COMMON_PARAMS["data.train_folds"] = [0, 1, 2]
 TRAIN_COMMON_PARAMS["data.validation_folds"] = [3]
-TRAIN_COMMON_PARAMS["data.samples_ids"] = FULL_GOLDEN_MEMBERS  # Change to None to use all members
+TRAIN_COMMON_PARAMS["data.samples_ids"] = {"all": None, "golden": FULL_GOLDEN_MEMBERS}["golden"]
 
 # ===============
 # PL Trainer
 # ===============
-TRAIN_COMMON_PARAMS["trainer.num_epochs"] = 20
+TRAIN_COMMON_PARAMS["trainer.num_epochs"] = 10
```

Comment:
10 epochs already achieve the best results?

Reply:
No. I set the runner for a quick run session (also the reason for `golden` as default.

I'm just about to deploy several final experiments and I'll set it as the best one.